src/bio/bio_xstream.c

```diff
@@ -901,16 +902,16 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id)
 
 		rc = spdk_conf_read(config, nvme_glb.bd_nvme_conf);
 		if (rc != 0) {
-			D_ERROR("failed to read %s, rc:%d\n",
-				nvme_glb.bd_nvme_conf, rc);
 			rc = -DER_INVAL; /* spdk_conf_read() returns -1 */
+			D_ERROR("failed to read %s, "DF_RC"\n",
+				nvme_glb.bd_nvme_conf, DP_RC(rc));
 			goto out;
 		}
 
 		if (spdk_conf_first_section(config) == NULL) {
-			D_ERROR("invalid format %s, rc:%d\n",
-				nvme_glb.bd_nvme_conf, rc);
 			rc = -DER_INVAL;
+			D_ERROR("invalid format %s, "DF_RC"\n",
```

Comment:
probably not much value here

Reply:
Agree, since there is no rc from spdk_conf_first_section(), it does not mean any real result of the call.